test/spmd/tensor/test_dtensor_ops.py

```diff
@@ -668,7 +669,15 @@ def wrapped(fn):
 }
 
 
+skip_bw = [None, "torch.isfinite"]
```

Comment:
what's None mean here?

Reply:
sorry, I should have added a comment. None is what func translates to for the transpose operations. There isn't a clean op name like torch.infinite.